bamboo/Dockerfile

```diff
@@ -1,6 +1,21 @@
 ## Dockerfile to create integration/unit test environment
 FROM node:10.16.3
+ADD ./bamboo/install_pyenv.sh /install_pyenv.sh
 RUN apt update && npm config set unsafe-perm true &&\
-    apt install -y netcat zip jq rsync zip python-pip && \
-    apt-get install -y python-pip && pip install awscli && \
-    npm install -g npm
\ No newline at end of file
+    apt install -y netcat zip jq rsync zip
+RUN apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
+libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+xz-utils tk-dev libffi-dev liblzma-dev
+RUN npm install -g npm
+RUN /install_pyenv.sh
+ENV PATH="/root/.pyenv/bin:$PATH"
```

Comment:
This series of commands updates the build container to utilize python 3.6.4 via pyenv.